packages/expo-cli/lib/help.js

```diff
@@ -16,4 +16,6 @@ module.exports = () => console.log(`
   options
     --project-root  set the path to the expo project
                     (defaults to the current working directory)
+    --npm           install packages using NPM
```

Comment:
```suggestion
    --npm           install packages using npm
```

nitpick but they seem to refer to themselves as "npm"